Mathlib/Order/InitialSeg.lean

```diff
@@ -10,16 +10,20 @@ import Mathlib.Data.Sum.Order
 /-!
 # Initial and principal segments
 
-This file defines initial and principal segments.
+This file defines initial and principal segment embeddings. Though these definitions make sense for
+arbitrary relations, they're intended for use with well orders.
+
+An initial segment is simply a lower set. A principal segment is a set of the form `Set.Iio x` for
```

Comment:
While "lower set" is a common enough term in order theory, I think we should still have something like the old description giving a quick explanation of the term. This saves the reader looking it up, and more importantly reassures that the term we use is the same as the informal one! (as you've just seen, that's not always the case in mathlib ;) )